ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/events/CreateTableEvent.java

```diff
@@ -88,7 +110,9 @@ private List<HivePrivilegeObject> getOutputHObjs() {
 
     COMMAND_STR = buildCommandString(COMMAND_STR,table);
 
-    LOG.debug("<== CreateTableEvent.getOutputHObjs(): ret={}", ret);
+    if (LOG.isDebugEnabled()) {
```

Comment:
minor, but why do we need this check?

Reply:
I think this is the format before. So I tried to keep the code in the same style.
Now I removed this check.